// @ngu/carousel/esm2015/lib/ngu-tile/ngu-tile.component.js
import { Component, HostBinding } from '@angular/core';
import * as i0 from "@angular/core";
export class NguTileComponent {
    constructor() {
        this.classes = true;
    }
}
NguTileComponent.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "12.2.13", ngImport: i0, type: NguTileComponent, deps: [], target: i0.ɵɵFactoryTarget.Component });
NguTileComponent.ɵcmp = i0.ɵɵngDeclareComponent({ minVersion: "12.0.0", version: "12.2.13", type: NguTileComponent, selector: "ngu-tile", host: { properties: { "class.item": "this.classes" } }, ngImport: i0, template: "<div class=\"tile\">\n  <ng-content></ng-content>\n</div>\n", styles: [":host{padding:10px;box-sizing:border-box}.tile{box-shadow:0 2px 5px #00000029,0 2px 10px #0000001f}\n"] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "12.2.13", ngImport: i0, type: NguTileComponent, decorators: [{
            type: Component,
            args: [{
                    selector: 'ngu-tile',
                    templateUrl: 'ngu-tile.component.html',
                    styleUrls: ['ngu-tile.component.scss']
                }]
        }], propDecorators: { classes: [{
                type: HostBinding,
                args: ['class.item']
            }] } });
