// @ngu/carousel/esm2015/lib/ngu-item/ngu-item.component.js
import { Component, HostBinding } from '@angular/core';
import * as i0 from "@angular/core";
export class NguItemComponent {
    constructor() {
        this.classes = true;
    }
}
NguItemComponent.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "12.2.13", ngImport: i0, type: NguItemComponent, deps: [], target: i0.ɵɵFactoryTarget.Component });
NguItemComponent.ɵcmp = i0.ɵɵngDeclareComponent({ minVersion: "12.0.0", version: "12.2.13", type: NguItemComponent, selector: "ngu-item", host: { properties: { "class.item": "this.classes" } }, ngImport: i0, template: "<ng-content></ng-content>\n", styles: [""] });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "12.2.13", ngImport: i0, type: NguItemComponent, decorators: [{
            type: Component,
            args: [{
                    selector: 'ngu-item',
                    templateUrl: 'ngu-item.component.html',
                    styleUrls: ['ngu-item.component.scss']
                }]
        }], propDecorators: { classes: [{
                type: HostBinding,
                args: ['class.item']
            }] } });
